tests/api_fastapi/core_api/routes/ui/test_structure.py

```diff
@@ -81,6 +132,28 @@ def make_dag(dag_maker, session, time_machine):
     dag_maker.sync_dagbag_to_db()
 
 
+@pytest.fixture
+def make_dag_with_multiple_version(dag_maker, session):
+    with dag_maker(DAG_ID) as dag:
+        EmptyOperator(task_id="task1")
+    dag.sync_to_db()
+    SerializedDagModel.write_dag(dag, bundle_name="dag_maker")
+    with dag_maker(DAG_ID) as dag2:
+        EmptyOperator(task_id="task1")
+        EmptyOperator(task_id="task2")
+    dag2.sync_to_db()
+    SerializedDagModel.write_dag(dag2, bundle_name="dag_maker")
+    with dag_maker(DAG_ID) as dag3:
+        EmptyOperator(task_id="task1")
+        EmptyOperator(task_id="task2")
+        EmptyOperator(task_id="task3")
+    dag3.sync_to_db()
+    SerializedDagModel.write_dag(dag3, bundle_name="dag_maker")
+
+    latest_version = DagVersion.get_latest_version(dag.dag_id)
```

Comment:
nit. Turning the duplicated works into one `create_dag_version` function.
```suggestion
    def create_dag_version(version: int) -> None:
        """Create a DAG with increasing number of tasks based on version number."""
        with dag_maker(DAG_ID) as dag:
            for i in range(1, version + 1):
                EmptyOperator(task_id=f"task{i}")
        dag.sync_to_db()
        SerializedDagModel.write_dag(dag, bundle_name="dag_maker")
        return dag

    for version in range(1, 4):
        dag = create_dag_version(version)

    latest_version = DagVersion.get_latest_version(dag.dag_id)
```

Reply:
Nice catch, thanks !